modules/axhal/src/arch/aarch64/context.rs

```diff
@@ -15,6 +15,145 @@ pub struct TrapFrame {
     pub spsr: u64,
 }
 
+impl TrapFrame {
+    /// Gets the 0th syscall argument.
+    pub const fn arg0(&self) -> usize {
```

Comment:
Can we just use `pub const fn arg(&self, index: usize) -> usize`?

Reply:
It may introduce overhead of index matching.